pkg/cred/cred.go

```diff
@@ -22,3 +24,20 @@ func GetCred() (*azidentity.DefaultAzureCredential, error) {
 
 	return cred, nil
 }
+
+func GetGraphCred() (*azidentity.DeviceCodeCredential, error) {
+	if graphCred != nil {
+		return graphCred, nil
+	}
+
+	gCreds, _ := azidentity.NewDeviceCodeCredential(&azidentity.DeviceCodeCredentialOptions{
+		TenantID: "", // todo - set this value
```

Comment:
are these supposed to still be blank with todos?

Reply:
I think I'm supposed to pass the values, I had to figure out how to pass them.